pkgs/development/python-modules/netbox-qrcode/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  pillow,
+  qrcode,
+  netbox,
+}:
+buildPythonPackage rec {
+  pname = "netbox-qrcode";
+  version = "0.0.15";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "netbox-community";
+    repo = "netbox-qrcode";
+    rev = "v${version}";
+    hash = "sha256-gEBOjmX7dcLjiy0u6raAtAaqZUVYsPz3YhupjRRqVDE=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+
+  propagatedBuildInputs = [
+    qrcode
+    pillow
+  ];
+
+  nativeCheckInputs = [ netbox ];
+
+  preFixup = ''
+    export PYTHONPATH=${netbox}/opt/netbox/netbox:$PYTHONPATH
+  '';
```

Comment:
pythonImportsCheck is missing. You probably want to copy the pattern from the other netbox plugins